prac_09/band.py

```diff
@@ -0,0 +1,26 @@
+"""CP1404 Week 09 Prac
+Band Class
+Estimated: 20 minutes
+Commenced 1:34pm
+Completed 2:03pm"""
+
+class Band():
```

Comment:
Great variable naming and formatting just remember to add docstrings for classes and methods.